lib/mistral_client/workflow.rb

```diff
@@ -25,9 +25,9 @@ def initialize(server, definition = nil, id: nil, name: nil)
     def execute!(env: nil, task_name: nil, input: nil)
       Execution.new(@server,
                     workflow_id: @id,
-                    env: env,
-                    task_name: task_name,
-                    input: input)
+                    env:,
+                    task_name:,
+                    input:)
```

Comment:
im confused by this. does this grab the value of the named parameter?
similar to just calling `super` in a child class? or just calling `raise` without additional params to reraise the exact same exception?

Reply:
Yeah, this is the ruby shorthand hash syntax, which apparently also applies to method calls. It was a default style setting in rubocop.

Reply:
this is pretty neat. available since 3.1